content/en/news/releases/1.9.x/announcing-1.9/_index.md

```diff
@@ -0,0 +1,117 @@
+### Virtual Machine (VM) expansion promoted to Beta
+
+Enabling workloads running in VMs to be part of the Istio service mesh, being able to apply consistent policy, and collect telemetry across containers and VMs has always been a focus of the Istio community.  We have continued improving the stability, testing and documentation for VM integration, and are happy to announce that in Istio 1.9 we have promoted this feature to Beta.
+
+Here's a list of supporting documents which you can follow to easily expand your Istio service mesh to include VMs:
+
+* [Virtual Machine Installation](/docs/setup/install/virtual-machine/) to get started.
+* [Virtual Machine Architecture](/docs/ops/deployment/vm-architecture/) to learn about the high level architecture of Istio's virtual machine integration.
+* [Debugging Virtual Machines](/docs/ops/diagnostic-tools/virtual-machines/) to learn more about troubleshooting issues with virtual machines.
+* [Bookinfo with a Virtual Machine](/docs/examples/virtual-machines/) to learn more about connecting virtual machine workloads to Kubernetes workloads.
+
+VM support will continue to be a high priority for us in 2021, and you can expect us to see more feature
+graduation in upcoming releases.
+
+### External Control Plane promoted to Beta
+
+As with VMs, management of external clusters is vital as Mesh Admins increasingly need to expand their service meshes
+to include remote clusters for many applications (e.g. global distribution, high availability). By promoting this
+feature to Beta we recognize this as a must for adoption and to promote widespread-usage.
+
+Similar to changes made for Virtual Machines, the Istio community has revamped and improved our
+[multicluster](/docs/setup/install/multicluster/) installation guides:
+
+* [Before you begin](/docs/setup/install/multicluster/before-you-begin/)
+* [Install Primary-Remote](/docs/setup/install/multicluster/primary-remote/)
+* [Install Primary-Remote on different networks](/docs/setup/install/multicluster/primary-remote_multi-network/)
+* [Verify installation](/docs/setup/install/multicluster/verify/)
+
+## `service-apis` promoted to Alpha
+
+Support for [Kubernetes `service-apis`](https://kubernetes-sigs.github.io/service-apis/) has been an active area of
+development since Istio 1.8. These are marketed as the next generation of the Ingress API, and draw inspiration from
+Istio's `Gateway` and `VirtualService`. The Istio community is eager to evolve with Kubernetes and these CRDs in upcoming
+releases. To try them out, check out the [getting started documentation](/docs/tasks/traffic-management/ingress/service-apis/).
+These APIs are still evolving, so feedback is welcome!
+
+## Better external authorization
+
+Authorization policy now supports an experimental feature of
+[`CUSTOM` action](/docs/reference/config/security/authorization-policy/#AuthorizationPolicy-Action) that allows users to
+integrate with external auth system (e.g. OPA, OAuth2, etc.) more easily.
```

Comment:
I suggest we give some background like:  If you are using envoy filter today to integrate with an external auth system, we would recommend to have you try this and give us the feedback!

Reply:
Added